RandomData.Api/Helpers/SwashbuckleHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
+
+namespace RandomData.Api.Helpers
+{
+    public static class SwashbuckleServiceHelper
+    {
+        public static IServiceCollection AddSwashbuckle(this IServiceCollection serviceCollection) =>
+            serviceCollection.AddSwaggerGen(setup =>
```

Comment:
Variable in this lambda is not "setup". Maybe "options" because type is SwaggerGenOptions.